packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -375,23 +375,31 @@ class RenderEditable extends RenderBox {
     return _textPainter.maxIntrinsicWidth;
   }
 
-  // This does not required the layout to be updated.
-  double get _preferredLineHeight => _textPainter.preferredLineHeight;
+  /// The height of a typical text line in logical pixels.
+  ///
+  /// This height is based on properties of the text's style such as
+  /// font size etc and could change when those properties change.
```

Comment:
Generically, the user wouldn't. It's mostly in line with the other getters in this file or that are downstream from the text painter like `text` or `textAlign`. 

In the context of its relation with the selection overlay, it seems like all property changes of this renderable are coordinated via the EditableTextState which creates the widgets that populates this renderable in the first place by _updateOrDisposeSelectionOverlayIfNeeded etc when anything changes.